test/grep.js

```diff
@@ -24,6 +24,7 @@ test('no args', t => {
   const result = shell.grep();
   t.truthy(shell.error());
   t.is(result.code, 2);
+  t.is(shell.errorCode(), 2);
```

Comment:
Can you also add a test to check when errorCode is 0?

Reply:
Thank you. Yes, I'll add that. Good thinking!

Reply:
Please let me know if the new test works. I'd also be happy to refactor the existing tests to use ```shell.errorCode()``` similar to how they use ```shell.error()``` instead of something like ```result.error```, but I know this would affect a lot of test files and wasn't sure if it would fit with this PR or with a new issue. In any case, thank you for your reviews, and I'm happy to make any changes, or add more tests.